bin/annotations_from_ensembl.sh

```diff
@@ -8,8 +8,8 @@ config=$1
 species=$(basename $config |  sed 's/\.[^ ]*//g')
 
 ## check env variables
-[ ! -z ${ENSEMBL_JSON_PATH+x} ] || (echo "Env var ENSEMBL_JSON_PATH not defined." && exit 1)
-[ ! -z ${OUTPUT_TSV_PATH+x} ] || (echo "Env var OUTPUT_TSV_PATH not defined." && exit 1)
+[ -z ${ENSEMBL_JSON_PATH+x} ] && echo "Env var ENSEMBL_JSON_PATH not defined." && exit 1
+[ -z ${ANNOTATIONS_PATH+x} ] && echo "Env var OUTPUT_TSV_PATH not defined." && exit 1
```

Comment:
Please change the echo message to be consistent. Was this mentioned in the readme as well? Is it consistent?

Reply:
@pcm32 I think you're looking at an outdated version for some reason, that should have been fixed.